src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -173,9 +173,14 @@ private void ViewLoadedHandler(object sender, RoutedEventArgs e)
         private void ViewModel_RequestZoomToFit(BoundingBox bounds)
         {
             var prevcamDir = watch_view.Camera.LookDirection;
-            watch_view.ZoomExtents(bounds.ToRect3D(.05));
+            var camera = watch_view.Camera as HelixToolkit.Wpf.SharpDX.PerspectiveCamera;
+
+            //Todo, Call the equivalent method in Helix on adoption of next release.
+            ZoomExtents(camera, (float)(watch_view.ActualWidth / watch_view.ActualHeight), bounds, out var pos, out var look, out var up);
+            camera.AnimateTo(pos.ToPoint3D(), look.ToVector3D(), up.ToVector3D(), 0);
```

Comment:
hmm, if you remove this line - does the camera update?
Also, neat? Is this worth setting to some value?

Reply:
Without this line then nothing happens.